TSOClient/tso.content/FSO.Content.csproj

```diff
@@ -176,6 +176,19 @@
     <Content Include="Content\Textures\EOD\Band\bandskilllevels.png">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="Content\Textures\EOD\Buzzer\eod_lightsframe1.png" />
```

Comment:
This content must be copied to the output directory.

Reply:
(and all others below it)

Reply:
I had some weird VS issues with this—it wanted to make 10000 changes by shifting everything down unnecessarily...so I must've messed up doing some line-item changes. 

Thanks for making these changes since time is more challenging now 👶